modal/runner.py

```diff
@@ -296,12 +295,8 @@ async def _run_app(
 
     output_mgr = _get_output_manager()
     if interactive and output_mgr is None:
-        warnings.warn(
-            "Interactive mode is disabled because no output manager is active. "
-            "Use 'with modal.enable_output():' to enable interactive mode and see logs.",
-            stacklevel=2,
-        )
-        interactive = False
+        msg = "Use the `modal.enable_output()` context manager to enable interactive mode."
```

Comment:
Phrasing is a little confusing here; setting `interactive=True` is what directly "enables" interactive mode but there's an additional condition that output must be enabled. Might articulate that more directly, like "Interactive mode requires output to be enabled (use the `modal.enable_output()` context manager)" or something.